include/tester/ssp/SSPAccount.hpp

```diff
@@ -79,9 +78,21 @@ class SSPAccount : public pj::Account
     {
         _call = call;
     }
-    
 
+    void call(std::string & id_to_call, std::string & domain)
+    {
+        _call = new SSPCall(*this);
+        std::string d_uri = "sip:" + id_to_call + "" + domain;
+        _call->callTo(d_uri);
+        //TODO: when to stop the call
+    }
+    
 private:
     SSPCall * _call;
     pj::AccountConfig _acc_cfg;
+
+    static constexpr auto sip = "sip:";
```

Comment:
Make these constexpr variables uppercase and also move them to be the top of the private block.